cmd/manager/main.go

```diff
@@ -115,6 +123,18 @@ func main() {
 		log.Info(err.Error())
 	}
 
+	metricsServer := osdmetrics.NewBuilder().
+		WithPort(osdMetricsPort).
+		WithPath(osdMetricsPath).
+		WithCollectors(localmetrics.MetricsList).
+		WithServiceName(operatorconfig.OperatorName).
```

Comment:
This will cause a conflict with the other metrics endpoint the operator-sdk deploys by default.

Reply:
Is that based off the operator name too?